strongbox-data-service/src/main/java/org/carlspring/strongbox/config/EmbeddedOrientDbConfig.java

```diff
@@ -28,9 +29,16 @@
 
     private static final Logger logger = LoggerFactory.getLogger(EmbeddedOrientDbConfig.class);
 
+    @Value("${strongbox.server.database.path:strongbox-vault/db}")
+    private String databasePath;
+
+    @Value("${strongbox.database.snapshot.resource:db-snapshot}")
```

Comment:
the default value here probably should be `classpath:/db-snapshot`?

Reply:
yes, thanks